magicctapipe/scripts/lst1_magic/semi_automatic_scripts/clusters.py

```diff
@@ -4,7 +4,7 @@
 __all__ = ["slurm_lines", "rc_lines"]
 
 
-def slurm_lines(queue, job_name, array=None, mem=None, out_name=None):
+def slurm_lines(queue, job_name, array=None, mem=None, out_name=None, nice_parameter=None):
```

Comment:
I'm not sure if None is a 'good' value for this parameter. Maybe the default should be 0 (i.e., no change in priorities). I have to check this but '#SBATCH --nice=None' seems quite anomalous

Reply:
In the config file default value is equal 0.